target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -134,6 +134,11 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan" "1:lan" "5:wan" "6@eth0"
 		;;
+	hootoo,tm05)
+		ucidef_add_switch "switch0"
+		ucidef_add_switch_attr "switch0" "enable" "false"
+		ucidef_set_interface_lan "eth0"
+		;;
```

Comment:
Please add the device to the existing case above.

Reply:
Sure, will do! I admit, had it separate to try different options - but makes sense, will combine. Thanks!